docs/specification-ligo.md

```diff
@@ -156,19 +156,19 @@ Each address can have any number of operators and be an operator of any number o
 ## Proposals
 
 Everyone can make a new proposal, however, you have to freeze some tokens for that.
-The proposer specifies how many tokens they want to freeze and this value is checked by
-the contract according to its compile-time configuration.
-If this value is accepted and the proposer has enough unfrozen tokens, these unfrozen tokens
-are frozen and voting starts.
+The proposer specifies how many frozen tokens they want to stake and this value is checked by
+the contract according to its compile-time configuration. Also tokens frozen in a voting period
+can only be used in the next period and further. This applies to proposal creation as well as to
+the `unfreeze` operation.
```

Comment:
```suggestion
The proposer specifies how many frozen tokens they want to stake and this value is checked by
the contract according to its compile-time configuration. 

This can only be performed in a proposing stage period, meaning one that's odd-numbered and
the proposer must have frozen his tokens in one of the preceding periods.
```